shared/templates/static/ansible/gnome_gdm_disable_guest_login.yml

```diff
@@ -0,0 +1,14 @@
+# platform = multi_platform_rhel, multi_platform_fedora
+# reboot = false
```

Comment:
doesn't this require reboot? gdm won't reload this file once it's already running AFAIK